src/DssDeploy.t.sol

```diff
@@ -34,7 +34,11 @@ contract DssDeployTest is DssDeployTestBase {
     function testJoinETH() public {
         deploy();
         assertEq(vat.gem("ETH", bytes32(bytes20(address(this)))), 0);
-        ethJoin.join.value(1 ether)(bytes32(bytes20(address(this))));
+        weth.join.value(1 ether)();
```

Comment:
The deployed weth has `deposit` / `withdraw` rather than `join` / `exit`, which are just here for ease in the tests.

Reply:
Fixed